api/HTMLInputElement.json

```
@@ -2201,6 +2201,57 @@
           }
         }
       },
+      "showPicker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/showPicker",
+          "spec_url": "https://html.spec.whatwg.org/multipage/input.html#dom-input-showpicker",
+          "support": {
+            "chrome": {
+              "version_added": "99"
+            },
+            "chrome_android": {
+              "version_added": "99"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Mozilla folks <a href='https://github.com/whatwg/html/pull/7319#issuecomment-988837778'>expressed interest</a>."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "Mozilla folks <a href='https://github.com/whatwg/html/pull/7319#issuecomment-988837778'>expressed interest</a>."
+            },
```

It's common to link to bugs in BCD, so let's do that instead:

```suggestion
            "firefox": {
              "version_added": false,
              "notes": "See <a href='https://bugzil.la/1745005'>bug 1745005</a>."
            },
            "firefox_android": {
              "version_added": false,
              "notes": "See <a href='https://bugzil.la/1745005'>bug 1745005</a>."
            },
```